app/routes/index.js

```diff
@@ -3,7 +3,24 @@ const router = express.Router();
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-    res.render('index', {title:'OG-YOGA'});
+    res.render('index', {title:'OG-YOGA', error: ""});
+});
+
+/* GET login page. */
+router.get('/login', function(req, res, next) {
+    res.render('../views/login');
```

Comment:
I don't think you have to render to ../views/login
just the following syntax will be fine

res.render('login', {title: 'Login'});